node-license-violations.json

```diff
@@ -0,0 +1,21 @@
+{
+  "deep": true,
+  "production": true,
+  "allow-licenses": [
+    "ISC",
+    "WTFPL",
+    "MIT",
+    "BSD",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "Apache-2.0",
+    "Unlicense",
```

Comment:
Are these extra licenses valid?  Either way I don't understand why we need two files.